client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -45,6 +48,10 @@ const refGoals: Record< string, Onboard.SiteGoal[] > = {
 	'create-blog-lp': [ SiteGoal.Write ],
 };
 
+const withLocale = ( url: string, locale: string ) => {
+	return locale && locale !== 'en' ? `${ url }/${ locale }` : url;
+};
```

Comment:
Does it make sense to use the `localizeUrl()` helper here? It should handle a few edge cases as well. Note that it has some extra args we should use carefully (like, whether the user is logged in)

Reply:
After a huddle @alshakero, he recommended me to use history.back() and this logic was removed.